src/i18n/json/fa-IR.json

```
@@ -13,6 +13,7 @@
   "Can work with DOM, run side effects, schedule updates.": "امکان خواندن DOM، اجرای موارد جانبی، زمان بندی آپدیت ها.",
   "React updates DOM and refs": "React آپدیت می کند DOM و ref ها را",
   "Read docs for {name} (opens in a new tab)": "برای مطالعه بیشتر درباره{name} (تب جدید باز می کند)",
+  "https://reactjs.org/docs/react-component.html#{docname}": "https://reactjs.org/docs/react-component.html#{docname}",
```

It needs contributors, but there is

```suggestion
  "https://reactjs.org/docs/react-component.html#{docname}": "https://fa.reactjs.org/docs/react-component.html#{docname}",
```